sys/vane/jael.hoon

```diff
@@ -1122,6 +1107,14 @@
       /(scot %p our)/vent-result
     ^-  note:able
     [%a %want [our who] /j/(scot %p our)/vent-result %vent-result res]
+  ::
+  ++  extract-snap                                    ::  extract rewind point
+    ^-  snapshot
+    :*  eve.urb
+        kyz.puk.sub
+        +.eth.sub
+        etn(source *(each ship node-src))
```

Comment:
This defaults to ~zod. We should arguably change the `each` around so it defaults to an "empty" `node-src` instead, but I'm not sure if this actually matters.

Reply:
We always replace `source.etn` in `++restore-snap`, so it doesn't matter in practice.  Arguably we should make our own little `++state-eth-node-snapshot` structure rather than keeping around dummy state like this.